openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -550,11 +551,24 @@ port_INLINE void activity_synchronize_endOfFrame(PORT_RADIOTIMER_WIDTH capturedT
       ieee154e_vars.dataReceived->l2_frameType = ieee802514_header.frameType;
       ieee154e_vars.dataReceived->l2_dsn       = ieee802514_header.dsn;
       memcpy(&(ieee154e_vars.dataReceived->l2_nextORpreviousHop),&(ieee802514_header.src),sizeof(open_addr_t));
-      
-      // toss the IEEE802.15.4 header -- this does not include IEs as they are processed 
+
+      // This is the case where we first hear an EB and do not yet know the ASN used

```

Comment:
hm, the new commits outdated the diff where you initially commented so it is hidden now. in any case, I addressed all the suggestions outside of ```IEEE802154_security.[ch]``` files, while @ssciancalepore is working on those!